tony-core/build.gradle

```diff
@@ -7,8 +7,8 @@ apply plugin: 'com.google.protobuf'
 dependencies {
   compile project(':tony-mini')
 
-  compile deps.azkaban.az_core
-  compile deps.azkaban.az_hadoop_jobtype_plugin
+  //compile deps.azkaban.az_core
+  //compile deps.azkaban.az_hadoop_jobtype_plugin
```

Comment:
Rather than remove these Azkaban dependencies, you can exclude the transitive Guava/Guice dependencies using something like this:

```
compile(deps.azkaban.az_core) {
  exclude group: 'com.google.guava'
  exclude group: 'com.google.inject'
  // alternatively, to be more specific
  // exclude group: 'com.google.inject', module 'guice'
}
```

Another command you can run to figure out what transitive dependencies are pulled in is:

```
./gradlew :tony-core:dependencies
```